snuba/datasets/factory.py

```diff
@@ -82,7 +82,7 @@ def get_enabled_dataset_names() -> Sequence[str]:
 
 
 def enforce_table_writer(dataset: Dataset) -> TableWriter:
```

Comment:
Could you please add a TODO to remove his method and mark it as deprecated? There should not be a default table writer for a dataset in a world with entities. It should be a writer per entity.